### PR TITLE
feat: remove suffix when only one page

### LINKED ISSUE #24: Skip use of page name in exported file when only one page
When an input file only contains one page, the filename can only be the name of the input file.

### PR DESCRIPTION
**Pull request by bogaertg**
* Add dockerignore to reduce docker context
* Add reference to build for step test in Makefil
* Skip PagePrefix when only one page in drawio file

Closes #24

### CODE REVIEW

File: runner.sh [hunk 1]
```diff
@@ -59,9 +59,16 @@ export_drawio_pages() {
   local filename="$3"
 
   local pagenum=0
+
+  local numberOfPage=$(sgrep -c '"name=\"".."\""' "$path")
```
Comment (rlespinasse) [inline runner.sh:63]: prefer the snake case notation for variable name

**Review by rlespinasse — CHANGES_REQUESTED** [review]
Can you add a flag to activate this feature (disable by default)?

Maybe 
```
--remove-page-suffix     Remove page suffix when possible
                         (in case of single page file)
```

Without this kind of flag, this pull request introduce a breaking change for drawio files with only one page.

File: runner.sh (hunk 1 above)
Comment (bogaertg) [reply runner.sh]: done

File: tests/tests.bats [hunk 2]
```diff
@@ -17,7 +17,7 @@
 }
 
 @test "Export as png using short options" {
-  docker_test "-E png -C -t -F test-assets-png" "" 0 "png"
+  docker_test "-E png --cli-options -t -F test-assets-png" "" 0 "png"
```
Comment (rlespinasse) [inline tests/tests.bats:20]: This changes seem not related to this PR purpose. Can you remove it?

**Review by rlespinasse — CHANGES_REQUESTED** [review]

File: tests/tests.bats (hunk 2 above)
Comment (bogaertg) [reply tests/tests.bats]: done

**Review by rlespinasse — APPROVED** [review]
